modules/nw-cluster-mtu-change.adoc

```diff
@@ -212,90 +212,56 @@ interface-name=enp1s0
 +
 [source,yaml]
 ----
-apiVersion: machineconfiguration.openshift.io/v1
-kind: MachineConfig
+variant: openshift
+version: 4.11.0
 metadata:
+  name: 01-control-plane-interface
   labels:
     machineconfiguration.openshift.io/role: master
-  name: 01-control-plane-interface
-spec:
-  config:
-    ignition:
-      config: {}
-      security:
-        tls: {}
-      timeouts: {}
-      version: 2.2.0
-    networkd: {}
-    passwd: {}
-    storage:
-      files:
-      - contents:
-          source: data:text/plain;charset=utf-8;base64,<encoded_config>
-          verification: {}
-        filesystem: root
-        mode: 420
-        path: /etc/NetworkManager/system-connections/<connection_name>
-    systemd: {}
+storage:
+  files:
+    - path: /etc/NetworkManager/system-connections/<connection_name> <1>
+      contents:
+        local: config.nmconnection <2>
+      mode: 0644
 ----
-+
---
-where:
-
-`<encoded_config>`:: Specifies the base64 encoding for the updated NetworkManager configuration.
-`<connection_name>`:: Specifies the NetworkManager connection name for the primary network interface.
---
+<1> Specify the NetworkManager connection name for the primary network interface.
+<2> Specify the local filename for the updated NetworkManager configuration file from the previous step.
 
-.... Create the following MachineConfig object in the `machine-config-worker.yaml` file:
+.... Create the following Butane config in the `worker-interface.bu` file:
 +
 [source,yaml]
 ----
-apiVersion: machineconfiguration.openshift.io/v1
-kind: MachineConfig
+variant: openshift
+version: 4.11.0
 metadata:
+  name: 01-worker-interface
   labels:
     machineconfiguration.openshift.io/role: worker
-  name: 01-worker-interface
-spec:
-  config:
-    ignition:
-      config: {}
-      security:
-        tls: {}
-      timeouts: {}
-      version: 2.2.0
-    networkd: {}
-    passwd: {}
-    storage:
-      files:
-      - contents:
-          source: data:text/plain;charset=utf-8;base64,<encoded_config>
-          verification: {}
-        filesystem: root
-        mode: 420
-        path: /etc/NetworkManager/system-connections/<connection_name>
-    systemd: {}
+storage:
+  files:
+    - path: /etc/NetworkManager/system-connections/<connection_name> <1>
+      contents:
+        local: config.nmconnection <2>
+      mode: 0644
 ----
-+
---
-where:
+<1> Specify the NetworkManager connection name for the primary network interface.
+<2> Specify the local filename for the updated NetworkManager configuration file from the previous step.
 
-`<encoded_config>`:: Specifies the base64 encoding for the updated NetworkManager configuration.
-`<connection_name>`:: Specifies the NetworkManager connection name for the primary network interface.
---
+.... Create `MachineConfig` objects from the Butane configs:
++
+[source,terminal]
+----
+$ for manifest in control-plane-interface worker-interface; do
```

Comment:
Not sure if the semicolon is a part of the command...should it be a period instead?

Reply:
No, the semicolon is correct.  It separates multiple commands on the same line; in this case, `for` and `do`.